examples/wesleycsj-helloWorld.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+ *	Author: WesleyCSJ
+ *  Description: Render four colors on screen with 2s delay
+ *
+ *
+ */
+#include <iostream>
+#include <SDL2/SDL.h>
+#include "GameLoop.hpp"
+#include "InputHandler.hpp"
+
+#define SCREEN_WIDTH  800
+#define SCREEN_HEIGHT 600
+
+std::map<const char*, SDL_Keycode> input_config = {
+    {"left", SDLK_LEFT},
+    {"right", SDLK_RIGHT},
+    {"quit", SDLK_ESCAPE},
+    {"double_size", SDLK_SPACE},
+    {"run", SDLK_LSHIFT}
+};
+
+class Game: public GameLoop {
+	using GameLoop::GameLoop;
+	InputHandler input_handler = InputHandler(input_config);
+	float screenCounter = 0;
+	int colorCursor = 0;
+	int colors[4][4] = {
+		{224,13 ,0  ,255},
+		{191,247,7  ,97 },
+		{55 ,224,0  ,255},
+		{0  ,1  ,224,255}
+	};
+	void start(){
+
```

Comment:
That is not necessary. You can delete it.

Reply:
do u mean start(), sure?

Reply:
Yes, start has a default empty implementation, so you don't need to put here if nothing to be started. Check by yourself:

https://github.com/NewayPix/hazard-memory/blob/master/src/GameLoop.hpp#L17-L21

start and stop has a empty implementation. However the other methods are abstract and should be implemented as you already did: render, update and event.

Reply:
Done.